### ISSUE TITLE
arcs-make tigmint command not found

### ISSUE THREAD

**Issue by mas160** [bcgsc/arcs#84]
hi,
I used arcs-make to run arcs with tigmint but I am getting the error : tigmint command not found from this line : 
$(gtime) tigmint tigmint draft=$(draft) reads=$(reads) minsize=$(minsize) as=$(as) nm=$(nm) dist=$(dist) mapq=$(mapq) trim=$(trim) span=$(span) window=$(window) t=$t 

Also can I run this arcs-make with multiple read files? if yes how is the command written?
Thank you

**Comment by lcoombe** [comment]
Hi @mas160,

Do you have the path to your `tigmint` installation on your PATH?
You can see by running `which tigmint`, and see if the path to your installation is listed.

At this time you cannot run `arcs-make` with multiple read files -- you need to concatenate your reads together first. You can get around this if you really want to by running the commands yourself instead of using the Makefile, but it just makes things a bit more complicated.

**Comment by mas160** [comment]
Hi,
Thank you for your reply.
Yes it is in my PATH so I added  tigmint-make file to my PATH and changed the command line to:
$(gtime) ./tigmint-make tigmint draft=$(draft) reads=$(reads) minsize=$(minsize) as=$(as) nm=$(nm) dist=$(dist) mapq=$(mapq) trim=$(trim) span=$(span) window=$(window) t=$t

it seems to be working for now...
I concatenated my reads but I am getting an error with gzread:
[gzread] fd:3: incorrect data check
[bam_sort_core] merging from 512 files and 8 in-memory blocks... time user=1178869.19s system=5098.15s elapsed=148254.18s cpu=798% memory=11128 job= time user=2190.34s system=599.57s elapsed=148254.18s cpu=1% memory=7 job= time user=40187.56s system=959.96s elapsed=167762.88s cpu=24% memory=6741 job=
make[1]: *** [vero1.all.sortbx.bam] Error 1
make[1]: *** Deleting file vero1.all.sortbx.bam' make[1]: Leaving directory ...'
make: *** [vero1.tigmint.fa] Error 2

**Comment by lcoombe** [comment]
It looks like there is something wrong with the format of your concatenated reads file. How did you do the concatenation?

You could run `gunzip --test <reads file>` to check the integrity of the file.

**Comment by mas160** [comment]
I did it and indeed the file was corrupted so I am concatenating it again. 
Concerning the run time: the arcs-make pipeline was running for almost two days before this error... What is the usual run time for 10x draft assembly of a 2.9 GB?

**Comment by lcoombe** [comment]
The runtime depends heavily on the read coverage you are using, as well as the number of threads. The main bottleneck is the `bwa mem` read alignments for both the Tigmint and ARCS steps. Based on the snippet of your log message, it looks like you didn't get past the initial alignment for Tigmint on your past run? 

I took at a past ARCS run we did in the group with 24 threads, and that took a bit less than 2 days. If you add Tigmint in before that, I'd estimate ~4 days? As I said, the vast majority of that time is spent in the alignment stage. If you want the ARCS step to go faster, I'd recommend using `ARKS` mode -- `arcs-make arks-tigmint`, which uses kmer matching instead of read alignments to map the barcoded reads to the contigs.
